src/VisualStudio/Core/Next/Progression/RoslynGraphProvider.cs

```diff
@@ -12,11 +11,11 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.CSharp.Progression
 {
-    [GraphProvider(Name = "CSharpRoslynProvider", IntellisenseType = Guids.CSharpProjectRootIdString)]
-    internal sealed class CSharpGraphProvider : AbstractGraphProvider
+    [GraphProvider(Name = "RoslynGraphProvider", ProjectCapability = "CSharp | VisualBasic")]
```

Comment:
The legacy project system also has the capability so a capability check works for both the new and the old project systems.